image/Dockerfile

```diff
@@ -2,7 +2,9 @@ FROM python:3.8-slim
 WORKDIR /app
 RUN mkdir -p /app
 COPY ./image/requirements.txt /app
-RUN pip install -r requirements.txt
+RUN   apt-get update -y && apt-get upgrade -y && apt-get install -y make
+RUN apt-get install  libglib2.0-dev python3-pip  python3-dev -y
```

Comment:
remove unnecessary packets

Reply:
when I removed the "apt-get install python3-pip" packet the pip install of bluepy returned error and failed to install it.

Reply:
what's the error?

Reply:
error: subprocess-exited-with-error
  
  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [13 lines of output]
      /usr/local/lib/python3.8/site-packages/setuptools/dist.py:697: UserWarning: Usage of dash-separated 'description-file' will not be supported in future versions. Please use the underscore name 'description_file' instead
        warnings.warn(
      running bdist_wheel
      running build
      running build_py
      Working dir is /tmp/pip-install-kx4ft85m/bluepy_d60630f289bb438b894af259dc13c020
      execute make -C ./bluepy clean
      execute make -C bluepy -j1
      Failed to compile bluepy-helper. Exiting install.
      Command was 'make -C bluepy -j1' in /tmp/pip-install-kx4ft85m/bluepy_d60630f289bb438b894af259dc13c020
      Return code was 2
      Output was:
      b"make: Entering directory '/tmp/pip-install-kx4ft85m/bluepy_d60630f289bb438b894af259dc13c020/bluepy'\ntar xzf bluez-src.tgz\ntouch ./bluez-5.47/lib/bluetooth.c ./bluez-5.47/lib/hci.c ./bluez-5.47/lib/sdp.c ./bluez-5.47/lib/uuid.c ./bluez-5.47/attrib/att.c ./bluez-5.47/attrib/gatt.c ./bluez-5.47/attrib/gattrib.c ./bluez-5.47/attrib/utils.c ./bluez-5.47/btio/btio.c ./bluez-5.47/src/log.c ./bluez-5.47/src/shared/mgmt.c ./bluez-5.47/src/shared/crypto.c ./bluez-5.47/src/shared/att.c ./bluez-5.47/src/shared/queue.c ./bluez-5.47/src/shared/util.c ./bluez-5.47/src/shared/io-glib.c ./bluez-5.47/src/shared/timeout-glib.c\ncc -L. -g -Wall  -Os -DHAVE_CONFIG_H -I./bluez-5.47/attrib -I./bluez-5.47 -I./bluez-5.47/lib -I./bluez-5.47/src -I./bluez-5.47/gdbus -I./bluez-5.47/btio -I./bluez-5.47/sys -I/usr/include/glib-2.0 -I/usr/lib/aarch64-linux-gnu/glib-2.0/include -o bluepy-helper bluepy-helper.c ./bluez-5.47/lib/bluetooth.c ./bluez-5.47/lib/hci.c ./bluez-5.47/lib/sdp.c ./bluez-5.47/lib/uuid.c ./bluez-5.47/attrib/att.c ./bluez-5.47/attrib/gatt.c ./bluez-5.47/attrib/gattrib.c ./bluez-5.47/attrib/utils.c ./bluez-5.47/btio/btio.c ./bluez-5.47/src/log.c ./bluez-5.47/src/shared/mgmt.c ./bluez-5.47/src/shared/crypto.c ./bluez-5.47/src/shared/att.c ./bluez-5.47/src/shared/queue.c ./bluez-5.47/src/shared/util.c ./bluez-5.47/src/shared/io-glib.c ./bluez-5.47/src/shared/timeout-glib.c -lglib-2.0\nmake: cc: No such file or directory\nmake: *** [Makefile:30: bluepy-helper] Error 127\nmake: Leaving directory '/tmp/pip-install-kx4ft85m/bluepy_d60630f289bb438b894af259dc13c020/bluepy'\n"
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for bluepy

Reply:
please look closely at this error message and try to figure out what the problem is

Reply:
I still figuring it out, looks bluepy-helper like didn't find 'nmake' and  

This error originates from a subprocess and is likely not a problem with pip.

above the error message said not a problem with pip, could be solved with install stable version of bluepy , if I am wrong to tell me to continue learning about that error